core/tests/puppeteer-acceptance-tests/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -17,9 +17,13 @@
  * tag properties from the blog-admin page.
  */
 
-// TODO(#16552): user-story(B8 cell) left as blocked by this issue.
-// TODO(#17162): This test can be done without the need of a super admin, as
-// blog admin can revoke blog editor role of other users from /blog-admin page.
+/** TODO(#16552): One user-story left when user cannot publish a blog post if
```

Comment:
TODO(#16552): Write a test that does XXX.

(You don't need "This test should be written as ..." at the end. That's implied by the issue number after the TODO.)

Reply:
done

Reply:
You haven't addressed the first part of my comment, which is to write the TODO as an imperative.

Reply:
@seanlip is it fine now?